index.js

```diff
@@ -42,46 +42,87 @@ module.exports.pitch = function (remainingRequest) {
   // This prevents [chunkhash] values from changing when running webpack
   // builds in different directories.
   // this.loadModule('@angular/core', (a,b,c,d) =>{debugger;});
-  if (this.query.exclude.some(mdl => this._module.rawRequest.match(new RegExp(mdl)))){
```

Comment:
why is this removed? it's important for excluding modules from being exposed

Reply:
fixed, added check for exclude object